From db21313ab55a1be3387d60890130cdb5f89b531c Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Fri, 16 Nov 2018 22:02:03 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3399pro-npu-evb-v10: Modify
 vop-bw-dmc-freq

Make the default configuration support 2048x1536 panel.

Change-Id: Idbcb6628b5697a067b3fa08be082019a1ae011f2
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399pro.dtsi | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399pro.dtsi b/arch/arm64/boot/dts/rockchip/rk3399pro.dtsi
index 1a295fcb94f4..a26f01e3ffa9 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399pro.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399pro.dtsi
@@ -38,9 +38,10 @@
 	>;
 	vop-bw-dmc-freq = <
 	/* min_bw(MB/s) max_bw(MB/s) freq(KHz) */
-		0       577      200000
-		578     1701     300000
-		1702    99999    400000
+		0       762      200000
+		763     1893     400000
+		1894    3012     528000
+		3013    99999    800000
 	>;
 	auto-min-freq = <200000>;
 };
-- 
2.35.3

